webCommander/centralHub/xTwitter/twitterAPI.py

```diff
@@ -44,6 +45,8 @@ def x_authenticate():
 def request_to_x_api(endpoint, headers={}, params={}):
     x_session = x_authenticate()
     response = x_session.get(endpoint, headers=headers, params=params)
+
+    print(f'Request to API {response.status_code}')
     while response.status_code == TOO_MANY_REQUESTS:
         print('Too many requests, trying in 15 mins...')
         sleep(900)
```

Comment:
this is a pretty big timeout, I think the web server will time out before then:

<img width="841" alt="image" src="https://github.com/abaddonpuff/commandCenter/assets/387927/3b90aa3d-5a23-456f-8e04-a13fd47e2528">